README.md

```diff
@@ -1,141 +1,82 @@
-        - containerPort: 8080
-        env:
-          - name: TERESAK8S_HOST
-            value: KUBERNETES_API_ENDPOINT
-          - name: TERESAK8S_USERNAME
-            value: KUBERNETES_API_USERNAME
-          - name: TERESAK8S_PASSWORD
-            value: KUBERNETES_API_PASSWORD
-          - name: TERESAK8S_INSECURE
-            value: "true"
-          - name: TERESAFILESTORAGE_TYPE
-            value: s3
-          - name: TERESAFILESTORAGE_AWS_KEY
-            value: AWS_ACCESS_KEY
-          - name: TERESAFILESTORAGE_AWS_SECRET
-            value: AWS_SECRET_KEY
-          - name: TERESAFILESTORAGE_AWS_REGION
-            value: AWS_REGION
-          - name: TERESAFILESTORAGE_AWS_BUCKET
-            value: S3_BUCKET
-          - name: TERESADEPLOY_REVISION_HISTORY_LIMIT
-            value: "5"
-          - name: TERESADEPLOY_START_TIMEOUT
-            value: 10m
-          - name: TERESADEPLOY_FINISH_TIMEOUT
-            value: 30m
-          - name: NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-```
-
-The TERESADEPLOY environment variables are optional.
-
-## View API Documentation
-
-To view the auto-generated swagger API documentation, the following command will compile, run a webserver and open your browser on the swagger-ui:
-
-    $ make swagger-docs
-
-## CLI
-
-Build the CLI:
-
-    $ make build-client
-
-Steps to setup a new cluster and deploy a new application, assuming you already have the API running:
+    $ export TERESA_RSA_PUBLIC=`cat teresa.rsa.pub | base64`
+    $ helm add repo luizalabs http://helm.k8s.magazineluiza.com
+    $ helm install luizalabs/teresa \
+        --set rsa.private=$TERESA_RSA_PRIVATE \
+        --set rsa.public=$TERESA_RSA_PUBLIC \
+        --set aws.key.access=xxxxxxxx \
+        --set aws.key.secret=xxxxxxxx \
+        --set aws.region=us-east-1 \
+        --set aws.s3.bucket=teresa \
+        --set db.name=teresa \
+        --set db.hostname=dbhostname \
+        --set db.username=teresa \
+        --set db.password=xxxxxxxx
+
+Look [here](./helm/README.md) for more information about helm options.
+
+## QuickStart
+
+Teresa has the concept of teams, which are just sets of users. An application
+belongs to a team and all its users can perform all operations on it. There are
+administrative users, which are just regular users with an admin flag set up and
+only them can do user and team management.
+
+To create an admin user you need access to the environment where the Teresa
+server is running (often a Kubernetes POD):
+
+    $ teresa-server create-super-user --email admin_email --password xxxxxxxx
+
+Now you can start creating other users and teams. First, you need to get the
+Teresa endpoint created by Kubernetes and configure the client (get it
+[here](https://github.com/luizalabs/teresa-api/releases/latest)):
 
     $ teresa config set-cluster mycluster --server <teresa-endpoint>
     $ teresa config use-cluster mycluster
```

Comment:
In the first `set-cluster` command, Teresa will set the cluster as the current cluster.
Maybe you can simplify to using only one command.